sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -55,15 +55,10 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-# These options have no dest and action is store_false in the
-# argparse and default is None. When parsing these options in a dict using
-# PipelineOptions,We either ignore/discard if these options are specified.
-# Defining a map with their dest would maintain consistency
-# across PipelineOptions(**dict), PipelineOptions.from_dictionary(dict),
-# and argparse.
-_STORE_FALSE_OPTIONS_WITH_DIFFERENT_DEST = {
-    'use_public_ips': 'no_use_public_ips'
-}
+# Map the boolean option with the flag_name for the flags that have a
```

Comment:
Consider the following for readability:

```
# Map of Boolean options to flag names for options that have a destination(dest)
# in parser.add_argument() different from the flag name 
# and whose default value  is `None`. 
_FLAG_THAT_SETS_FALSE_VALUE =  {...}
```

Reply:
Variable name sounds good. Changed the comments a little bit